jboss-eap-cd-logging/added/logging.properties

```diff
@@ -32,7 +32,7 @@ handlers=CONSOLE
 
 handler.CONSOLE=org.jboss.logmanager.handlers.ConsoleHandler
 handler.CONSOLE.level=ALL
-handler.CONSOLE.formatter=COLOR-PATTERN
+handler.CONSOLE.formatter=##CONSOLE-FORMATTER##
```

Comment:
This still bothers me a bit, as this file is going to get copied in during build and then it's during launch that the sed happens. And unless that sed happens the file is broken.

I should probably just get over it as it's the same thing as we do here: https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss-eap-cd-openshift-launch/added/standalone-openshift.xml#L177

:)

I was going to suggest just dropping the ## business and just having the sed replace the whole line, but then if the user brought their own logging.properties we'd be modifying it even if they didn't want that.

Reply:
This now includes json_logging.sh as well, which will run in both cases. I don't have any objection to replacing the whole line either, say having it default to:

handler.CONSOLE.formatter=COLOR-PATTERN

But, yeah, agreed on the default datasources thing too.